### PR TITLE
com_menus. External link. Add support for protocols viber and skype

### PR DESCRIPTION
**Pull request by ghost**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/15534

### Summary of Changes
Add support for uri schemes/protocols `skype` and `viber` for field `link` in menu item type `external link`.

### Testing Instructions
- Create new menu item of type External link.
- Try to save links like
skype:vasya
skype://vasya
viber:+38099999999
viber://+38099999999 
- You'll get a warning "Save not permitted."

- Apply patch.

- Try again

### Comments
To be discussed? 
@brianteeman mentioned correctly "skype is only in the provisional list of supported uri schema in the standards and viber is not in the list at all" at https://www.iana.org/assignments/uri-schemes/uri-schemes.xhtml in his comment https://github.com/joomla/joomla-cms/issues/15534#issuecomment-297044083

But see my comment, too: https://github.com/joomla/joomla-cms/issues/15534#issuecomment-297081362

So, I don't know...

### CODE REVIEW

File: administrator/components/com_menus/controllers/item.php [hunk 1]
```diff
@@ -331,7 +331,7 @@ public function save($key = null, $urlVar = null)
 				$segments = explode(':', $data['link']);
 				$protocol = strtolower($segments[0]);
 				$scheme = array('http', 'https', 'ftp', 'ftps', 'gopher', 'mailto', 'news', 'prospero', 'telnet', 'rlogin', 'tn3270', 'wais', 'url',
-					'mid', 'cid', 'nntp', 'tel', 'urn', 'ldap', 'file', 'fax', 'modem', 'git', 'sms');
+					'mid', 'cid', 'nntp', 'tel', 'urn', 'ldap', 'file', 'fax', 'modem', 'git', 'sms', 'skype', 'viber',);
```
Comment (Quy) [inline administrator/components/com_menus/controllers/item.php]: Remove the last comma.

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 887b41f94a88886f2d6de2732e49577deb69da1f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15537">issues.joomla.org/tracker/joomla-cms/15537</a>.</sub>

**Comment by brianteeman** [comment]
I am not in favour of this PR.

**Comment by izharaazmi** [comment]
I don't see the use case of having these links in **menu**, (except my own [homepage](http;//izharaazmi.com)) 😊. I have similar views for few already there in the list.

No objection though.

**Comment by bertmert** [comment]
Closing because of lack of interest from issue opener since 4 weeks.

**Pull request closed without merging**